internal/datastore/common/sql.go

```diff
@@ -663,7 +713,11 @@ func (exc QueryRelationshipsExecutor) ExecuteQuery(
 			return nil, datastore.ErrCursorsWithoutSorting
 		}
 
-		query = query.After(queryOpts.After, queryOpts.Sort)
+		q, err := query.After(queryOpts.After, queryOpts.Sort)
```

Comment:
no unit test coverage for this

Reply:
Not sure what you're pointing to here

Reply:
sql_test has a bunch of tests for `After` and sort..?

Reply:
run the code coverage in `sql_test` and you will see